admin/load.php

```diff
@@ -613,12 +639,80 @@ function perflab_deactivate_plugin() {
  * Callback function hooked to admin_notices to render plugin activate/deactivate notices.
  *
  * @since n.e.x.t
- *
- * @return void
  */
 function perflab_plugin_admin_notices() {
+	$standalone_plugins_with_active_modules = perflab_get_standalone_plugins_with_active_modules();
 	if ( 'settings_page_perflab-modules' !== get_current_screen()->id ) {
-		return;
+		if ( ! empty( $standalone_plugins_with_active_modules ) ) {
+			echo '<div class="notice notice-warning is-dismissible"><p>';
+				printf(
+					/* translators: Link to Performance Lab Settings screen */
+					esc_html__( 'Performance Lab has detected usage of modules that have standalone plugin equivalents. Please visit the %1$s screen to see the plugins available for installation.', 'performance-lab' ),
+					sprintf(
+						'<a href="%s">%s</a>',
+						esc_html( menu_page_url( PERFLAB_MODULES_SCREEN, false ) ),
```

Comment:
```suggestion
						esc_url( menu_page_url( PERFLAB_MODULES_SCREEN, false ) ),
```

Reply:
Just removed the other page admin notice so not needed now.